src/sentry/static/sentry/app/stores/groupingStore.jsx

```diff
@@ -58,39 +65,14 @@ const GroupingStore = Reflux.createStore({
     });
   },
 
-  // Resets status of a remaining item (to be unmerged) if it exists
-  resetRemainingUnmergeItem() {
-    if (!this.remainingItem) return;
-
-    // If there was a single unchecked item before, make sure we reset its disabled state
-    this.setStateForId(this.unmergeState, this.remainingItem.id, {
-      disabled: false
-    });
-    this.remainingItem = null;
-  },
-
-  checkForRemainingUnmergeItem() {
+  isAllUnmergedSelected() {
     let lockedItems = Array.from(this.unmergeState.values()).filter(({busy}) => busy) || [
     ];
-    let hasRemainingItem =
-      this.unmergeList.size + 1 === this.mergedItems.length - lockedItems.length;
-
-    if (!hasRemainingItem) return;
-
-    // Check if there's only one remaining item, and make sure to disable it from being
-    // selected to unmerge
-    let remainingItem = this.mergedItems.find(item => {
-      let notSelected = !this.unmergeList.has(item.id);
-      let itemState = this.unmergeState.has(item.id) && this.unmergeState.get(item.id);
-      return notSelected && (!itemState || !itemState.busy);
-    });
-
-    if (!remainingItem) return;
-
-    this.remainingItem = remainingItem;
-    this.setStateForId(this.unmergeState, remainingItem.id, {
-      disabled: true
-    });
+    return (
+      this.unmergeList.size ===
+      this.mergedItems.filter(({latestEvent}) => !!latestEvent).length -
```

Comment:
do you need the `!!` here? or do you just like the semantic hinting that it's being used as a bool?

Reply:
It's unnecessary, I do like the semantic hinting, and it's a habit. Does it help or hinder reading at all?

Reply:
na, it's totally neutral in my eyes, probably wasn't worth commenting